pkg/client/client_test.go

```diff
@@ -276,14 +279,17 @@ func TestIndexes(t *testing.T) {
 
 	c, _ := New(opts)
 
-	r, co, err := c.indexes("products")
+	r, co, err := c.constraints("products")
+
+	t.Logf("constraints columns %v", co)
+	t.Logf("constraints content %v", r)
```

Comment:
These logs are not needed. If the test fail, you will get the info. If it doesn't fail, I don't see a need to see these lines when running a `go test`

Reply:
Let's create an issue to improve the test suite.